src/test/app/forms/validation/validators/IsLessThanOrEqualToSumOf.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai'
+import { IsLessThanOrEqualToSumOfConstraint } from 'forms/validation/validators/isLessThanOrEqualToSumOf'
+import { ValidationArguments } from 'class-validator'
+
+const DEFAULT_FIELD_TO_SUM_VALUE = 30
+const DEFAULT_VALUE = 200
+
+function validationArgs (fieldToSumValue: number, value: number): ValidationArguments {
+  return {
+    value: undefined,
+    targetName: undefined,
+    object: { fieldToSum: fieldToSumValue, relatedField: value },
+    property: undefined,
+    constraints: ['fieldToSum', 'relatedField']
+  }
+}
+
+describe('IsLessThanConstraint', () => {
```

Comment:
Will be easier for use if name of the test block reflects tested class. In that case it would be `IsLessThanOrEqualToSumOfConstraint`.

Reply:
oops, thought I got everything